evm_trace/__init__.py

```diff
@@ -1,3 +1,10 @@
 from .base import CallTreeNode, CallType, TraceFrame, get_calltree_from_trace
+from .parity import get_calltree_from_parity_trace
 
-__all__ = ["CallTreeNode", "CallType", "get_calltree_from_trace", "TraceFrame"]
+__all__ = [
+    "CallTreeNode",
+    "CallType",
+    "get_calltree_from_trace",
```

Comment:
should we rename to `get_calltree_from_geth_trace` for consistency?

Reply:
Yeah